scalafix/input/src/main/scala/fix/CanBuildFromSrc.scala

```diff
@@ -26,8 +26,11 @@ class CanBuildFromSrc() {
     b.result()
   }
 
-  // negative test
-  // def g[C0, A, C1[_]](c0: C0)(implicit cbf3: CanBuildFrom[C0, A, C1[A]]): C1[A] = {
-  //   cbf3().result()
-  // }
+  def kind(implicit cbf: CanBuildFrom[String, Char, String],
+                    cbf2: CanBuildFrom[String, (Int, Boolean), Map[Int, Boolean]]): Unit = {
+
+    cbf("")
+    cbf2("")
```

Comment:
Can you include a `.to` call?

Reply:
`.to` takes one type parameter
https://github.com/scala/scala/blob/v2.12.6/src/library/scala/collection/TraversableOnce.scala#L308

Reply:
Yes, so that means that it’s actually impossible to use `.to` with `String` or `Map`… So, I think this test is complete.